### PR TITLE
pwsafe: 1.08.2 -> 1.09.0

### PR DESCRIPTION
**Pull request by chrbauer**
fixes the broken build
there is a problem with wxGTK 3.1.2
maybe related to
https://github.com/pwsafe/pwsafe/blob/master/src/ui/wxWidgets/TreeCtrl.cpp
line 107
So I use wxGTK30

file is a new depedency

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by chrbauer** [comment]
I'm using this version for some time on nixos 20.03. The good thing is: the build is working.
But there are some issues: 
DnD in Gnome crashes the gnome session and copy password to clipboard has a problem. :-(

**Comment by chrbauer** [comment]
DnD problem doesn't occur with KDE

**Comment by mrkkrp** [comment]
Is there something that prevents this from being merged?

**Comment by chrbauer** [comment]
the current status is: it builds again and you have access to your passwords. but there are some bugs in this app. i don't have the time to investigate.

**Comment by chrbauer** [comment]
i guess it should be merged to 20.03 as well

File: pkgs/applications/misc/pwsafe/default.nix [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
 
   nativeBuildInputs = [ 
     cmake gettext perl pkgconfig zip
   ];
   buildInputs = [
-    libXext libXi libXt libXtst wxGTK31
+    libXext libXi libXt libXtst wxGTK30
     curl qrencode libuuid openssl xercesc
     libyubikey yubikey-personalization
+    file
```
Comment (Mic92) [inline pkgs/applications/misc/pwsafe/default.nix]: This likely belongs to `nativeBuildInputs` if it is needed only at build time.

Comment (chrbauer) [reply pkgs/applications/misc/pwsafe/default.nix]: not the file executable was required but a header file  like include/magic.h

Comment (chrbauer) [reply pkgs/applications/misc/pwsafe/default.nix]: ldd pwsafe: 	libmagic.so.1 => /nix/store/d7wqn6pn23m6ijj4qxmnyya7313vx4lc-file-5.38/lib/libmagic.so.1 (0x00007fc146ca9000)

Comment (Mic92) [reply pkgs/applications/misc/pwsafe/default.nix]: fair.

**Comment by Mic92** [comment]
backport: 3f90dcb0751

**Pull request merged**